docs/components.md

```diff
@@ -150,6 +150,32 @@ Kibana connects to the Elasticsearch cluster via the `elasticsearch-logging` Kub
 
 Kibana is a stateless component and therefore does not have any persistent storage requirements.
 
+#### Customization
+
+##### Kibana plug-ins
+
+Add-on functionality for Kibana is implemented with plug-in modules. Some known plug-ins for Kibana are listed here: https://www.elastic.co/guide/en/kibana/6.7/known-plugins.html.
+
+To install additional plug-ins, override the `plugins` item inside the `kibana` scope, like this:
+
+```jsonnet
+(import "../../manifests/platforms/aks.jsonnet") {
+    config:: import "kubeprod-autogen.json",
+    kibana+: {
+        plugins:: {
```

Comment:
Just in case we need to install our own plugins at some point:

```suggestion
        plugins+: {
```

Reply:
Done